src/index.html

```diff
@@ -13,6 +13,40 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar__startDay--sun calendar__monthLength--31">
+      <div class="calendar__space">
+        <div class="calendar__day calendar__day--1"></div>
+        <div class="calendar__day calendar__day--2"></div>
+        <div class="calendar__day calendar__day--3"></div>
+        <div class="calendar__day calendar__day--4"></div>
```

Comment:
not hard code every value. Use `nth-child` in ur loop. Remove all such classes
```suggestion
        <div class="calendar__day"></div>
        <div class="calendar__day"></div>
        <div class="calendar__day"></div>
        <div class="calendar__day"></div>
```